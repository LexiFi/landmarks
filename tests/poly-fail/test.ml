let ok =
  let test = (fun x -> x)
  in test "string", test 1;;

let ko =
  let test = (fun x -> x)[@landmark "test"]
  in test "string", test 1;;
