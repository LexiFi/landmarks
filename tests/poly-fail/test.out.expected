val ok : string * int = ("string", 1)
Line 4, characters 25-26:
4 |   in test "string", test 1;;
                             ^
Error: This expression has type "int" but an expression was expected of type
         "string"

